Update styling of enrolled button and add a check mark

noisecapella opened the pull request:
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Tag @ferdi or @pdpinch for review
- [x] Testing
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Fixes #721 

#### What's this PR do?
Fixes styling of enrolled buttons to look like the mockup. Note that the check mark is slightly different. I used the SVG from open-iconic. Should we look for a narrower check mark?

#### How should this be manually tested?
Go to the program or course run page for a product which you have purchased, and for another product you have not. Verify that they look similar to the screenshots below

#### Screenshots
Enrolled:
![Screenshot from 2019-07-03 14-42-19](https://user-images.githubusercontent.com/863262/60617028-c8297880-9da0-11e9-8ce5-87b08b62b8de.png)

Enroll now (should look the same as before):
![Screenshot from 2019-07-03 14-43-16](https://user-images.githubusercontent.com/863262/60617089-eb542800-9da0-11e9-92d8-becacb05a84b.png)

pdpinch commented:
I'd like to put off merging this until Monday. 

It's fine to review, just don't merge.

gsidebo commented:
@noisecapella We have a few SVG icons copied over from material.io (`expand_less` and `expand_more`, for example). Those icons are Apache 2.0 licensed. They also have a check mark icon: https://material.io/tools/icons/?icon=check&style=baseline

So two questions:
1. Should we add a license file for the material icons? Seems like we should, but if you think that's best to add in a separate issue, that's OK
1. Could we just use the material.io check mark to avoid using icons from multiple different libraries?

noisecapella commented:
@gsidebo Makes sense, I'll switch to material.io. I'll copy over the license file as well

noisecapella commented:
@gsidebo I switched to material.io but I refactored it to use the web fonts instead of self hosted svgs. Let me know what you think. Here's the check box:

![Screenshot from 2019-07-10 11-50-21](https://user-images.githubusercontent.com/863262/60985156-f7913580-a30a-11e9-8950-007d36db8ee9.png)

pdpinch commented:
I thought we were using bootstrap?

gsidebo commented:
@pdpinch This is just for SVG icons (and fonts, I guess). Bootstrap doesn't have its own icon library.

gsidebo commented:
@pdpinch FYI, we show the "Enroll Now" button to users that are enrolled in the given course, but in some run that isn't the unexpired run closest to the current date. That means if a user is enrolled in some future run, and there's some other unexpired run that happens before it, we show the "Enroll Now" button. That behavior existed for this PR. Is that what we want?

pdpinch commented:
Frankly, that sounds confusing. 

Can we fix that in another PR?

> @pdpinch <https://github.com/pdpinch> FYI, we show the "Enroll Now" button to users that are enrolled in the given course, but in some run that isn't the unexpired run closest to the current date. That means if a user is enrolled in some future run, and there's some other unexpired run that happens before it, we show the "Enroll Now" button. That behavior existed for this PR. Is that what we want?
